pymc3/sampling.py

```diff
@@ -111,31 +111,31 @@ def instantiate_steppers(_model, steps, selected_steps, step_kwargs=None):
     ----------
     model : Model object
         A fully-specified model object; legacy argument -- ignored
-    steps : step function or vector of step functions
+    steps : sequence of step functions
         One or more step functions that have been assigned to some subset of
-        the model's parameters. Defaults to None (no assigned variables).
+        the model's parameters.
     selected_steps : dictionary of step methods and variables
-        The step methods and the variables that have were assigned to them.
+        The step methods and the (possibly zero) variables that have were assigned to them.
```

Comment:
And here, it should say
```
selected_steps : dict
    A dictionary that maps a step method class to a list of model variables.
```

Reply:
Sure - I've added `zero or more` here because it's the fact that there could be zero _here_ which was causing the `continue` line to be uncovered